### PR TITLE
fix: [Fix #59] Implement mobile navbar close menu

### LINKED ISSUE #59: [BUG] Hide close button in mobile-menu after click hambarger button
**Describe the bug**
When we click Hamburger button to navigate nav-links after clicks close button shown hide

**To Reproduce**
1. Go to mobile-menu
2. Click on Hamburger
3. Shown hide close button

Untitled

### PR DESCRIPTION
**Pull request by simrin3308**
## What does this PR do?

This PR implements the mobile nav menu which was earlier notshowing.

Fixes #59

BEFORE=>
image

AFTER=>
image

## How should this be tested?

1. Open the website locally http://localhost:3000/
2. Go To inspect element and decrease the screen size when hamburger menu is visible.
3. click on the hamburger menu.
4. You will see close menu on the top right of the screen

Thank You

### CODE REVIEW

**Comment by TheMohit2003** [comment]
Hey @simrin3308, I also had a similar test failure. This is what Piyush gave as a solution in [this comment](https://github.com/piyushgarg-dev/piyushgargdev-nextjs/pull/63#issuecomment-1580116459). 

Updated title :     **[Fix #59] Implement mobile navbar close menu**  
  
Copy the above title text and paste it by editing the above pull request title .
image

**Comment by simrin3308** [comment]
@TheMohit2003 Thank you for the help. I tried to do so but its still failing...

**Comment by TheMohit2003** [comment]
@simrin3308 , try this : `fix: [Fix #59] Implement mobile navbar close menu`

**Comment by TheMohit2003** [comment]
A brief summary on what went wrong :     

The previous pull request title, "[Fix #59] Implement mobile navbar close menu," was missing some important information. When creating a pull request, it's important to follow a specific format to make it easier for others to understand what changes are being made. In this case, the pull request title was missing a prefix that tells us what type of change is being made. This prefix helps categorize the change and gives a quick idea of what the pull request is about.

To fix the issue, we need to add the correct prefix to the pull request title. In this case, since it's a bug fix, the prefix "fix:" should be added. So the new pull request title would be "fix: [Fix #59] Implement mobile navbar close menu." This way, people reviewing the pull request will immediately know that it's a bug fix and can focus on checking if the changes address the issue properly.  

Refer piyush's [reply](https://github.com/piyushgarg-dev/piyushgargdev-nextjs/pull/63#issuecomment-1580116459)

**Comment by TheMohit2003** [comment]
hey @piyushgarg-dev, why is the vercel test failing? The pull request test failed before but after changes to the title, it has now passed. Even to know what is the error, I clicked on the details of vercel  test but this is what the webpage is showing me instead of the deployment error : 
image

**Comment by simrin3308** [comment]
Hey @TheMohit2003. My Vercel test was also failing. I had one commit that didn`t do anything(dummy commit). I removed that commit and all worked good. It's just a workaround. But I don't know the actual cause. Let me know if you found anything. Thank you

**Comment by simrin3308** [comment]
@TheMohit2003 Its still failing.

**Comment by TheMohit2003** [comment]
@simrin3308 , Piyush is going to go live today at [9:00 PM](https://www.youtube.com/watch?v=mMl_Eegy3bQ). I will try asking him there , as I have already tagged him in one of the above comment

**Comment by simrin3308** [comment]
@piyushgarg-dev Thank you for the quick response. Feel free to to ask me if i need to do something related to this RR. Thank you

**Comment by piyushgarg-dev** [comment]
Also, Please remove the package-lock.json file as we are using yarn.lock file. These files conflict with each other.
Thanks :)

**Comment by simrin3308** [comment]
Hey @piyushgarg-dev. I have removed the package-lock.json file as instructed. Thank You:)

**Comment by piyushgarg-dev** [comment]
Can you install the `react-icons` and push the package.json file ✨
@simrin3308

File: components/Header/Header.jsx [hunk 1]
```diff
@@ -71,6 +75,10 @@ const Header = () => {
             onClick={toggleMenu}
           >
             <div className={`${classes.nav__menu}`}>
+              {crossMenu && <div className="border  text-3xl  absolute top-10 right-10 font-extrabold
```
Comment (piyushgarg-dev) [inline components/Header/Header.jsx:78]: Hey @simrin3308, Let's remove the border here 🙌🏻

**Comment by simrin3308** [comment]
@piyushgarg-dev I have installed `react-icons` and also pushed the update. Do review and share suggestions. Thank You.

**Pull request merged**